.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+offerings:
+# below is an example of a Deployable Architecture (DA) solution
+- name: IBM Cloud HPC (DA) # must match the offering name in the ibm_catalog.json
```

Comment:
This is the offering programmatic name. In your ibm_catalog.json its set to `terraform-ibm-modules-terraform-ibm-hpc-ad6e71e` - these need to match

Reply:
updated